html/captive-portal/lib/captiveportal/PacketFence/Controller/Status.pm

```diff
@@ -45,20 +46,38 @@ sub index : Path : Args(0) {
     } else {
         $c->forward('userIsNotAuthenticated');
     }
+    if (person_has_local_account($pid)) {
+        $c->stash->{hasLocalAccount} = 1;
+    }
     $c->stash(
-        title => "State - Network Access",
+        title => "Status - Network Access",
         template => 'status.html',
         billing  => $c->profile->hasBilling(),
         access_registration_when_registered => $c->profile->canAccessRegistrationWhenRegistered(),
     );
 }
 
+sub has_open_violation : Path : Args(1) {
```

Comment:
This checks for lost/stolen violation only so change it to is_lost_stolen so its clearer